backend/config/initializers/devise.rb

```diff
@@ -312,4 +312,11 @@
   # When set to false, does not sign a user in automatically after their password is
   # changed. Defaults to true, so a user is signed in automatically after changing a password.
   # config.sign_in_after_change_password = true
+
+  config.jwt do |jwt|
+    jwt.secret = Rails.application.credentials.jwt_secret_key
+    jwt.dispatch_requests = [['POST', %r{^/login$}]]
+    jwt.revocation_requests = [['DELETE', %r{^/logout$}]]
+    jwt.expiration_time = 1.day.to_i
```

Comment:
Minor point – is there any advantage to having a 1 day expiry time? Surveyors will mostly go out in evenings, so maybe having a 6 hour expiry would be better than the default of 1 hour, but 1 day means it will probably be expiring as they start the next day, but still leave the tokens valid longer in a less useful way.

Reply:
I think I'm going to change this to 365 days because signing in is annoying. Thanks for flagging.